src/Rule/UrlHandler.php

```diff
@@ -55,13 +55,28 @@ public function validate(mixed $value, object $rule, ValidationContext $context)
         return $result;
     }
 
+    /**
```

Comment:
I'd mention here that it's just a wrapper over `idn_to_ascii` built-in function with a different handling of return value. A `{@link}` / `{@see}` tag to original function won't be redundant too in my opinion.